databases/sql-server/prisma/schema.prisma

```diff
@@ -1,9 +1,46 @@
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["microsoftSqlServer", "connectOrCreate"]
+  previewFeatures = ["microsoftSqlServer", "referentialActions"]
```

Comment:
Depending on when we want to merge we might skip the microsoft feature flag but for now that's fine